pkgs/sanposhiho/gomockhandler/registry.yaml

```diff
@@ -0,0 +1,6 @@
+packages:
+  - type: go_install
+    repo_owner: sanposhiho
+    repo_name: gomockhandler
+    description: Mr. gomockhandler is the clever and more agile manager of golang/mock
+    version_source: github_tag
```

Comment:
I don't think this is needed because this tool has GitHub Relesases.

https://github.com/sanposhiho/gomockhandler/releases

https://aquaproj.github.io/docs/reference/registry-config/version-source